docs/pages/web/popover.js

```diff
@@ -1022,7 +1021,7 @@ function ScrollBoundaryContainerExample() {
                     flex="grow"
                   >
                     <Button
```

Comment:
I think it's another color

![Screen Shot 2022-12-05 at 2 32 09 PM](https://user-images.githubusercontent.com/10593890/205726307-de97e83b-2bc6-4314-9312-e2222284e20c.png)